Gemfile

```diff
@@ -9,7 +9,7 @@ gem 'pry'
 # Prevents bundler from taking a long-time to resolve
 group :development, :test do
   gem 'activerecord-import'
-  gem 'mime-types', '~> 1.16'
+  gem 'mime-types', '>= 1.16', '< 4'
```

Comment:
Done with a slightly different change. I have copied the `mime-types` restriction to the Appraisal so that, as you prefer, you can start varying these for each version of Rails + AWS supported by Paperclip.